page_objects/menu_page.js

```diff
@@ -0,0 +1,53 @@
+let BasePage = require("../base/base.page");
+let Button = require("../elements/button.element")
+
+let womenTabLocator = by.xpath('//a[.="Women"]')
+let productLocator = by.xpath('(//a[.="Summer Dresses"])[1]')
+let addtocart1Locator = by.xpath('(//span[contains(text(),"Add to cart")])[1]')
```

Comment:
`addtocart1Locator, addtocart2Locator, addtocart3Locator` - use single locator and element object

Reply:
Fixed